QualityControl/public/common/runModeHeader.js

```diff
@@ -0,0 +1,236 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { RunStatus } from '../common/enums/RunStatus.enum.js';
+import { h, info } from '/js/src/index.js';
+
+/**
+ * Run mode header component showing run information and exit button
+ * @param {Model} model - root model of the application
+ * @param {FilterModel} filterModel - model that manages filter state
+ * @param {PageModel} pageModel - model that manages the state of the page
+ * @returns {vnode} - virtual node element
+ */
+export function runModeHeader(model, filterModel, pageModel) {
+  if (!model.inRunMode) {
+    return null;
+  }
+
+  return h('.run-mode-header.flex-row.items-center.p2.g2.bg-gray-lighter', {
+    id: 'run-mode-header',
+  }, [
+    renderRunModeInfo(model, filterModel),
+    h('.flex-grow'),
+    renderExitButton(filterModel, pageModel),
+  ]);
+
+  /**
+   * Renders the run mode information section
+   * @param {Model} model - root model of the application
+   * @param {FilterModel} filterModel - model that manages filter state
+   * @returns {vnode} - virtual node element
+   */
+  function renderRunModeInfo(model, filterModel) {
+    return h('.run-mode-info.flex-row.items-center.g2', {
+      id: 'run-mode-info',
+    }, [
+      renderTitle(),
+      renderRunNumber(model.runNumber),
```

Comment:
RunNumber is already provided by the FilterModel. Could we not use it from there and remove this getter from the Model.js?